test/node/extended_json.test.ts

```diff
@@ -300,14 +300,16 @@ describe('Extended JSON', function () {
     expect(serialized).to.equal('{"a":10}');
   });
 
-  it('should throw if invalid BSON types are input to EJSON serializer', function () {
+  it.skip('should throw if invalid BSON types are input to EJSON serializer', function () {
+    // TODO This doesn't throw for the reason you'd expect it to
```

Comment:
Needs JIRA ticket, EJSON does not have the same assertions BSON does, this was passing for the wrong reason.

Reply:
https://jira.mongodb.org/browse/NODE-4952